Gemfile.lock

```diff
@@ -659,7 +659,7 @@ GEM
     recaptcha (5.17.0)
     redcarpet (3.6.0)
     redis (4.8.1)
-    redis-actionpack (5.4.0)
+    redis-actionpack (5.3.0)
```

Comment:
This need to be `5.4.0` for bundle to run

Reply:
Thanks, just fixed that.